src/catalog/database_catalog.cpp

```diff
@@ -1813,6 +1813,10 @@ void DatabaseCatalog::BootstrapProcs(const common::ManagedPointer<transaction::T
                   postgres::NAMESPACE_DEFAULT_NAMESPACE_OID, {}, {}, {}, {}, str_type, "", false);
   // TODO(tanujnay112): no op codes for lower and upper yet
 
+  // Extract Year
+  CreateProcedure(txn, postgres::EXTRACT_YEAR_PRO_OID, "extractYear", postgres::INTERNAL_LANGUAGE_OID,
```

Comment:
Does this SQL function work? If yes, then please add tests to [functions.test](https://github.com/cmu-db/terrier/blob/master/script/testing/junit/traces/functions.test)

Reply:
Not exactly. I have posted a separte issue #1068